test_runner/src/main/kotlin/ftl/gc/GcIosTestMatrix.kt

```diff
@@ -62,6 +63,7 @@ object GcIosTestMatrix {
 
         val iOSTestSetup = IosTestSetup()
             .setNetworkProfile(args.networkProfile)
+            .setPullDirectories(args.directoriesToPull.toIosDeviceFiles())
```

Comment:
Interesting, for android we pass a list of strings... and the field itself is named differently `directoriesToPull` 😸

Reply:
google API is full of surprises